ssl_checker.py

```diff
@@ -0,0 +1,68 @@
+import socket
+import ssl
+from datetime import datetime, timedelta
+from typing import Dict, Union
+
+import telegram_send
+
+DAYS_TO_EXPIRATION = 50
+SHOW = True
+
+
+def check_cert_validity(cert: Dict) -> bool:
```

Comment:
missed docstrings

Reply:
also, it's a good practise to specify types of k/v pairs inside a dict, f.e. Dict[str, str]